api/bibliotheca.py

```diff
@@ -534,7 +541,7 @@ def findaway_license_to_webpub_manifest(
             license document via Bibliotheca, or a dictionary
             representing such a document loaded into JSON form.
         """
-        if isinstance(findaway_license, basestring):
+        if isinstance(findaway_license, (bytes, unicode)):
```

Comment:
[The porting guide](https://portingguide.readthedocs.io/en/latest/strings.html?highlight=__repr__#type-checking) mentions `six.string_types` as a substitute for basestring, but it doesn't do what we want in this case, because in Python 3, `bytes` isn't considered a "string type".